README.md

```diff
@@ -199,13 +199,13 @@ Supported clients are:
 * [php-http-304](https://alexandre.alapetite.fr/doc-alex/php-http-304/)
 * [jQuery](https://jquery.com/)
 * [lib_opml](https://github.com/marienfressinaud/lib_opml)
-* [jQuery Plugin Sticky-Kit](https://leafo.net/sticky-kit/)
 * [keyboard_shortcuts](http://www.openjs.com/scripts/events/keyboard_shortcuts/)
 * [flotr2](http://www.humblesoftware.com/flotr2)
 
-## Only for some options
+## Only for some options or configurations
```

Comment:
Only for optional features?

Reply:
It is meant to cover e.g.:
* the case of the JavaScript scrollbar, which is only for browsers without native support
* the case of bcrypt.js, only for login using Web form (login is not an optional feature, but there are a few different options, only one of which requires bcrypt.js

Reply:
It tickles my stylistic sensitivities, but it'll do. ;-)